content/blog/writing-crossguard-policies/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Writing CrossGuard Policies"
```

Comment:
Nit: Can we jazz this up any? "Enforcing different kinds of policies for cloud resources"?  "Writing X policies" sounds super generic. While name-dropping "CrossGuard" is nice, it might not be catchy.